cmd/translatesfx/translatesfx/otel.go

```diff
@@ -33,28 +33,33 @@ func saInfoToOtelConfig(cfg saCfgInfo) otelCfg {
 			receivers[k.(string)] = v
 		}
 	}
+	const rd = "resourcedetection"
 	out := otelCfg{
 		ConfigSources: map[string]interface{}{
 			// TODO check if should be nil
 			"include": nil,
 		},
 		Receivers: receivers,
+		Processors: map[string]interface{}{
+			rd: map[string]interface{}{
+				"detectors": []string{"system", "env"},
```

Comment:
I wonder if we should add the cloud provider resource detectors for generating their respective unique IDs (`AWSUniqueId`, `gcp_id`, `azure_resource_id`)?

Reply:
Hmm not sure -- at this point I'm targeting/automating the stuff that [this document](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/signalfx-smart-agent-migration.md) recommends.

Thoughts @rmfitzpatrick?

Reply:
I think the end goal for the tool is not just 1:1 config content but also all-around agent experience.  Believe the agent detects the cloud provider w/ its hostid utility, which adding all the resource detector targets would effectively do as well.  My concern would be the chattiness and continued noise w/ failed detection attempts though.

Reply:
I think we should be aligned with the default config where we have all of them added by default https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/agent_config.yaml#L106

Reply:
Thanks guys for the feedback. I've added "gce", "ecs", "ec2", and "azure" resource detectors.